### ISSUE TITLE
Exception when attempting to login: "TypeError: Cannot read property 'sort' of undefined

### ISSUE THREAD

**Issue by toadhats** [Alex-Rose/fb-messenger-cli#107]
I'm getting an intermittent issue attempting to launch the application. I recall also seeing the same error the other day when attempting to switch conversations, but it resolved itself. I don't remember doing anything specific to resolve it that time, or last time I saw this issue when attempting to login. I'm running it in cmd, on Windows 10 (latest updates etc). 

Stack trace:
```
TypeError: Cannot read property 'sort' of undefined
    at C:\Users\*****\AppData\Roaming\npm\node_modules\fb-messenger-cli\scripts\listeners.js:34:12
    at Request._callback (C:\Users\*****\AppData\Roaming\npm\node_modules\fb-messenger-cli\scripts\messenger.js:400:9)
    at Request.self.callback (C:\Users\*****\AppData\Roaming\npm\node_modules\fb-messenger-cli\node_modules\request\request.js:188:22)
    at emitTwo (events.js:125:13)
    at Request.emit (events.js:213:7)
    at Request.<anonymous> (C:\Users\*****\AppData\Roaming\npm\node_modules\fb-messenger-cli\node_modules\request\request.js:1171:10)
    at emitOne (events.js:115:13)
    at Request.emit (events.js:210:7)
    at Gunzip.<anonymous> (C:\Users\*****\AppData\Roaming\npm\node_modules\fb-messenger-cli\node_modules\request\request.js:1091:12)
    at Object.onceWrapper (events.js:314:30)
```

After 3 further attempts just then, i was able to log in successfully, and the app is now working without any apparent issues.

**Comment by toadhats** [comment]
Not sure if this is a duplicate of #83, apologies if that's the case

**Comment by SamBergeron** [comment]
Hey, this error refers to 
```TypeError: Cannot read property 'sort' of undefined
    at C:\Users\*****\AppData\Roaming\npm\node_modules\fb-messenger-cli\scripts\listeners.js:34:12```
and sort is no longer at that line.
Are you using the latest version of the code? I vaguely remember someone else getting the same thing but I think we fixed it.
You can update with npm or just pull the latest code from here.

**Comment by toadhats** [comment]
I thought I was on the latest, but it looks like I'm not. I'll update from 1.7 to 1.8 and see if I get this issue again.

**Comment by SamBergeron** [comment]
This is definitely fixed in 4f1faa2701e8d5442332a1388488fb13036c30f1 it was caused by trying to sort an invalid number of conversations. I've fixed the settings so that if for whatever reason, the number of conversations to load isn't there it'll set it to the default 15 (and also save it to your settings)
